nautobot/docs/user-guide/core-data-model/ipam/namespace.md

```diff
@@ -0,0 +1,14 @@
+# Namespaces
+
++++ 2.0.0
+
+Each namespace has a name and a description, and can optionally be associated to a location for informational purposes.
+
+A namespace groups together a set of related but distinct [VRFs](vrf.md), [prefixes](prefix.md), and [IP addresses](ipaddress.md). Fundamentally, its purpose is to serve as a constraint or boundary for uniqueness and duplication of IPAM data. If you have a relatively straightforward network, where there are no overlapping prefixes or duplicated IP addresses, a single namespace may be sufficient to model your entire network, even if it has thousands of IPAM records; however, in the case of a managed service provider network or similar, you may need multiple namespaces to accurately model its complexities and clearly distinguish between otherwise seemingly-duplicate records.
+
+Within a given namespace, only a single record may exist for each distinct VRF, prefix, or IP address. Although such a record may be used in multiple locations within your network, such as a VRF being configured on multiple devices, or a virtual IP address being assigned to multiple interfaces or devices, it is fundamentally a single network object in these cases, and Nautobot models the data.
```

Comment:
> and Nautobot models the data.

Was there supposed to be more here?

Reply:
Looks like I lost my train of thought, yeah. Perhaps "..and Nautobot models this data accordingly?"

Reply:
That's what I was thinking too